### PR TITLE
Fix #1592 - Add priority labels for ranking

### LINKED ISSUE #1592: Create 3 labels for issue priority from alexa ranking
This is the a part of #1533, we need to map the result from #1579 to a label. Label name should follow the bug label naming style. Some ideas:
Prefixes: priority, alexa, ranking, etc.
Suffixes: (critical, important, normal), (1, 2, 3), etc. 
Maybe we need a vote function on github to decide the name?

- [ ] Whitelist priority-* in label-list.js
- [ ] Add labels in web-bugs repository

### PR DESCRIPTION
**Pull request by MDTsai**
1. Whitelist "priority"-prefix label in label-list.js
2. Only one "priority"-prefix label is checked in labels.js
   Extend "status"-prefix label's implementation

r?= @miketaylr or you could suggest others to review :)

### CODE REVIEW

**Comment by MDTsai** [comment]
r? = @magsout

File: webcompat/static/js/lib/labels.js [hunk 1]
```diff
@@ -150,15 +150,16 @@ issues.LabelEditorView = Backbone.View.extend({
...
-    // If the change event comes from a "status"-type label,
-    // enumerate all checked "status"-type labels and uncheck
+    // We try to make sure only one "status" or "priority"-type label is set
+    // If the change event comes from a "status" or "priority"-type label,
+    // enumerate all checked "status" or "priority"-type labels and uncheck
     // the others.
     var checked;
     if (
-      $(evt.target).data("remotename").match(/^status/) && evt.target.checked
+      $(evt.target).data("remotename").match(/^(status|priority)/) && evt.target.checked
```
Comment (magsout) [inline webcompat/static/js/lib/labels.js]: Just a tiny comment. May be we can declare a var for `$(evt.target).data("remote name")` that we use twice.

**Comment by magsout** [comment]
Except my tiny comment, looks good.

@MDTsai r+

**Comment by MDTsai** [comment]
Update as @magsout 's comment. Ready for merge.

**Comment by magsout** [comment]
@MDTsai awesome.

Could you run: `npm run fix` and commit,  to fix the failed on travis

**Comment by magsout** [comment]
@MDTsai https://travis-ci.org/webcompat/webcompat.com/builds/248227183#L693

**Pull request merged**